app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -134,9 +134,10 @@ def raw_status
     raise MiqException::MiqOrchestrationStatusError, err.to_s, err.backtrace
   end
 
-  def raw_stdout
+  def raw_stdout(format = nil)
```

Comment:
You could make it a keyword arg, and then a literal nil passed in will get defaulted (I think)